app/Jobs/ApplicationDeploymentJob.php

```
@@ -1288,24 +1285,34 @@ private function elixir_finetunes()
         }
+        if (! $nixpacks_php_root_dir) {
+            $nixpacks_php_root_dir = new EnvironmentVariable;
+            $nixpacks_php_root_dir->key = 'NIXPACKS_PHP_ROOT_DIR';
+            $nixpacks_php_root_dir->value = '/app/public';
+            $nixpacks_php_root_dir->resourceable_id = $this->application->id;
+            $nixpacks_php_root_dir->resourceable_type = 'App\Models\Application';
+            $nixpacks_php_root_dir->save();
+        }
+
+        return [$nixpacks_php_fallback_path, $nixpacks_php_root_dir];
     }
```

PHPMD (2.15.0)

The method laravel_finetunes is not named in camelCase. (undefined)

(CamelCaseMethodName)